src/content/en/fundamentals/performance/speed-tools/index.md

```diff
@@ -0,0 +1,334 @@
+    </p>
+  </div>
+</div>
+<div class="text-and-image">
+  <img src="images/network-icons.svg" alt="A collection of network iconography representing different devices and network conditions.">
+  <div>
+    <h3>MYTH 2</h3>
+    <p>
+      <strong>User experience can be captured with a single “representative user.”</strong><br>
+      Real-world performance is highly variable due to differences in users’ devices, network performance, and other factors. Calibrate your lab and development environment to test a variety of such different conditions. Use field data to inform selection of test parameters for device type (i.e. mobile vs desktop), network connections (i.e. 3G or 4G), and other key variables.
+    </p>
+  </div>
+</div>
+<div class="text-and-image">
+  <img src="images/users.svg" alt="An assortment of depictions of different kinds of users.">
+  <div>
+    <h3>MYTH 3</h3>
+    <p>
+      <strong>My website loads fast for me, so it should load fast for my users.</strong><br>
+      The devices and networks that developers test load performance on are often much faster than what your users actually experience. Use field data to understand what form factors and networks your users are on and appropriately mirror those conditions when you test performance.
+    </p>
+  </div>
+</div>
+
+## Understanding lab vs. field data
+
+### Lab data
+
+<img src="images/tech.svg" class="attempt-right" alt="A graphical depiction of several people getting around and interacting in a busy public setting while using mobile devices.">
+
+When performance data is collected within a controlled environment with predefined device and network settings. This offers reproducible results and debugging capabilities to help identify, isolate, and fix performance issues.
+
+#### Strengths
+
+- Helpful for debugging performance issues
+- End-to-end and deep visibility into the user experience
+- Reproducible testing and debugging environment
+
+#### Limitations
+
+- Might not capture real-world bottlenecks
+- Cannot correlate against real-world page KPIs
+
+Note: Tools like [Lighthouse](/web/tools/lighthouse/) and [WebPageTest](https://www.webpagetest.org/) collect this.
+
+### Field data
+
+<img src="images/on-the-street.svg" class="attempt-right" alt="A graphical depiction of several people getting around and interacting in a busy public setting while using mobile devices.">
+
+When you collect performance data from real page loads your users are experiencing in the wild. Also called Real User Monitoring (or RUM).
+
+#### Strengths
+
+- Captures true real-world user experience
+- Enables correlation to business key performance indicators
+
+#### Limitations
+
+- Restricted set of metrics
+- Limited debugging capabilities
+
+Note: Tools like Speed Scorecard, [Chrome UX Report](/web/tools/chrome-user-experience-report/), and the Page Speed Insights speed score report this.
+
+## What are the different performance tools?
+
+<div class="tools">
```

Comment:
Please use standard DevSite styles here. For example - change to:
```
### Lighthouse {: .myth }

![lighthouse](images/tool-lighthouse.svg){: .attempt-left .myth }

Gives you personalized advice on how to improve your website across
performance, accessibility, PWA, SEO, and other best practices.

<div class="clearfix"></div>
```

Reply:
Wrt the DevSite styles Pete's referring to, he mentioned our blue boxes are very similar to the ones at the bottom of https://developers.google.com/web/ and perhaps we should reuse the styling for those.